.github/workflows/static-analysis.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        ocp-version: [ 'dev-master' ]
+        ocp-version: [ 'dev-stable26' ]
```

Comment:
dev-stable26 does not seems to exists in the nextcloud/ocp package, should we keep `dev-master` until we have a 26.x available ?

Reply:
It is (now) available: https://packagist.org/packages/nextcloud/ocp#dev-stable26